.github/workflows/tests.yml

```diff
@@ -0,0 +1,64 @@
+name: PHPUnit
+
+on:
+  push:
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  tests:
+    name: Tests
+    runs-on: Ubuntu-20.04
+
+    strategy:
+      matrix:
+        include:
+          - php: '5.4'
+          - php: '5.5'
+          - php: '5.6'
+          - php: '7.0'
+          - php: '7.1'
+          - php: '7.2'
+          - php: '7.3'
+          - php: '7.4'
+          - php: '8.0'
+          - php: '8.1'
```

Comment:
The old CI setup also had a job running tests with the min stability of composer set to `dev`

Reply:
Added that for 7.4 and 8.0